### DISCUSSION TITLE
Updating from version 0.7.0

### DISCUSSION THREAD

**Discussion by IsabellaElectrum** [gssapi/gssproxy#90] [Q&A] [answered]
Hello, we are currently running version 0.7.0 and would like to update to the newest version 0.9.2 Our management has asked us to confirm that performing this upgrade will not break the configuration due to making a large jump in versions.

**Comment by oerdnj** [comment]
That’s something only your own integration testing can tell you.

**Comment by simo5** [comment, marked as answer]
All I can tell you is that we do not make intentional breaking changes if we can help it, this project take ABI/API forward compatibility quite seriously.
But you should test the change yourself to make sure, there is always a chance of an unintentional bug or an undefined side effect that you rely on changing behavior.

**Reply by IsabellaElectrum** [reply]
Thank you for your help!
